doc-content/enterprise-only/openshift/offline-repo-proc.adoc

```diff
@@ -1,16 +1,54 @@
 [id='offline-repo-proc']
-= Preparing a Maven mirror repository for offline use 
+= Preparing a Maven mirror repository for offline use
 
 If your {OPENSHIFT} environment does not have outgoing access to the public Internet, you must prepare a Maven repository with a mirror of all the necessary artifacts and make this repository available to your environment.
 
 You do not need to complete this procedure if your {OPENSHIFT} environment is connected to the Internet.
 
-.Procedure
+.Prerequisites
 
-. Prepare a Maven release repository to which you can write. The repository must allow read access without authentication. Your OpenShift environment must have access to this repository. You can deploy a Nexus repository manager in the OpenShift environment. For instructions about setting up Nexus on OpenShift, see https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/developer_guide/tutorials#nexus-setting-up-nexus[Setting up Nexus]. 
+* You have a computer that has outgoing access to the public Internet.
 
-. Complete the following actions on a computer that has an outgoing connection to the public Internet:
+.Procedure
 
+ifeval::["{context}"=="openshift-immutable"]
+. Prepare a Maven release repository to which you can write. The repository must allow read access without authentication. Your OpenShift environment must have access to this repository. You can deploy a Nexus repository manager in the OpenShift environment. For instructions about setting up Nexus on OpenShift, see https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/developer_guide/tutorials#nexus-setting-up-nexus[Setting up Nexus]. Use this repository as a mirror repository.
+ifdef::PAM[]
+If you are planning to create immutable servers from KJAR services or to deploy {CENTRAL} Monitoring,
+endif::PAM[]
+ifdef::DM[]
+If you are planning to create immutable servers from KJAR services,
+endif::DM[]
+place your services in this repository as well. You must configure this repository as the external Maven repository. You cannot configure a separate mirror repository in an immutable environment.
+endif::[]
+ifeval::["{context}"!="openshift-immutable"]
+. Prepare a Maven release repository to which you can write. The repository must allow read access without authentication. Your OpenShift environment must have access to this repository. You can deploy a Nexus repository manager in the OpenShift environment. For instructions about setting up Nexus on OpenShift, see https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/developer_guide/tutorials#nexus-setting-up-nexus[Setting up Nexus]. This repository is to be used as a separate mirror repository.
++
+Alternatively, if you use a custom external repository (for example, Nexus) for your services, you can use the same repository as a mirror repository.
```

Comment:
"This repository is to be used as a separate mirror repository." Passive voice.
Can we say instead: Use this repository as a separate mirror repository".